mvc/controllers/dependency-injection.rst

```diff
@@ -1,8 +1,87 @@
-.. include:: /../common/stub-topic.txt
+Dependency Injection and Controllers
+====================================
 
-|stub-icon| Dependency Injection and Controllers
-================================================
+By `Steve Smith`_
 
-.. include:: /../common/stub-notice.txt
+ASP.NET MVC 6 controllers should request their dependencies explicitly via their constructors. In some cases, though, it may be preferable to populate dependencies using property injection, which is also supported (for controllers).
```

Comment:
Property injection is no longer supported by ASP.NET's built-in DI. Only ctor injection, and on MVC controller action methods.